### PR TITLE
Enable runtime tests with closed issues

### PR DESCRIPTION
**Pull request by EgorBo**
I used a simple script to remove tests from targets.issues with links to already closed github issues.
(will run stress tests after the normal run)

### CODE REVIEW

File: src/tests/issues.targets [hunk 1]
```diff
@@ -2143,10 +1825,10 @@
             <Issue>needs triage</Issue>
         </ExcludeList>
         <ExcludeList Include = "$(XunitTestBinBase)/JIT/Methodical/eh/basics/throwinfilter_d/**">
-            <Issue>https://github.com/dotnet/runtime/issues/54389</Issue>
+            <Issue>https://github.com/dotnet/runtime/issues/47624</Issue>
         </ExcludeList>
         <ExcludeList Include = "$(XunitTestBinBase)/JIT/Methodical/eh/basics/throwinfilter_r/**">
-            <Issue>https://github.com/dotnet/runtime/issues/54389</Issue>
+            <Issue>https://github.com/dotnet/runtime/issues/47624</Issue>
         </ExcludeList>
```
Comment (EgorBo) [inline src/tests/issues.targets:1832]: Adjusted links here (because original ones were closed as duplicates)

**Comment by danmoseley** [comment]
We should at some point do an exercise like this for libraries tests.

**Comment by EgorBo** [comment]
https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/String.Manipulation.cs#L1710-L1732

**Comment by EgorBo** [comment]
@dotnet/jit-contrib PTAL

**Review by jakobbotsch — APPROVED** [review]

**Pull request merged**